jetson/percep/pcl.cpp

```diff
@@ -310,39 +310,28 @@ double PCL::FindClearPath(const std::vector<std::vector<int>> &interest_points)
 
     //Check Center Path
     if(CheckPath(interest_points, obstacles, compareLine(0,-HALF_ROVER), compareLine(0,HALF_ROVER))) {
-        #if PERCEPTION_DEBUG
+      leftBearing = 0; // When no obstacles detected, reset bearings
+      rightBearing = 0;
+      #if PERCEPTION_DEBUG
             std::cout << "CENTER PATH IS CLEAR!!!" << std::endl;
-        #endif
-        return 0;
+      #endif
     }
-    else { //otherwise, check to find left and right paths
-
-        //Values that store the distances of the last obstacle from a given CheckPath. Center value gets its distance from previous loop of CheckPath
-        double centerDistance = distance, leftDistance, rightDistance;
+    
+    else {
 
         //Initialize base cases outside of scope
         vector<int> centerObstacles = {obstacles.at(0), obstacles.at(1)};
 
-        //Find Clear left path
-        double leftAngle = getAngleOffCenter(10, 0, interest_points, obstacles);
-        leftDistance = distance; 
-
+        //Find Clear left path, set left bearing
+        leftBearing = getAngleOffCenter(10, 0, interest_points, obstacles);
+        
         //Reset global variables
         obstacles = {0, 0};
         obstacles.at(0) = centerObstacles.at(0);
         obstacles.at(1) = centerObstacles.at(1);
 
-        //Find clear right path
```

Comment:
I believe that the lines you deleted here are Kevin's depth code. Could you add that back in?